source/common/protobuf/utility.cc

```diff
@@ -209,4 +209,10 @@ uint64_t DurationUtil::durationToSeconds(const ProtobufWkt::Duration& duration)
   return Protobuf::util::TimeUtil::DurationToSeconds(duration);
 }
 
+void DurationUtil::writeSystemClockTime(const SystemTime system_clock_time,
```

Comment:
Probably best to put this in `TimestampUtil` as it relates to `Timestamp` rather than `Duration`.